src/app/plugins/connectfour.plugin.ts

```diff
@@ -144,6 +144,11 @@ class ConnectFourGame {
   private _playingLion: boolean;
   private _aiDepth = 4;
 
+  // messy initialization since typescript won't let me use -1 or null
+  private _timeoutId: ReturnType<typeof setTimeout> = setTimeout(function () {});
+
+  private _moveTimeLimit = 2 * 60 * 1000; // milliseconds
```

Comment:
what's this time limit in seconds? it'd be more helpful to have that number as a comment IMO

Reply:
We have a module in the package called [ms](https://github.com/cs-discord-at-ucf/lion/blob/03a650e26f1e4afc07dee0f93d1ec0a289d7d807/src/app/handlers/command.handler.ts#L74)

Make it a lot more readable

Reply:
Definitely. I changed it to use ms instead.